### PR TITLE
Cathy says hello

### PR DESCRIPTION
**Pull request by cathy-french**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+			Console.WriteLine("Hello from Cathy");
```
Comment (ChrisRSE) [inline Program.cs:10]: Testing comment..

**Review by ChrisRSE — APPROVED** [review]
Works great! Ignore what **Atchu** says 😈

**Review by atchu98 — CHANGES_REQUESTED** [review]
Please fix code

File: Program.cs (hunk 1 above)
Comment (cathy-french) [reply Program.cs:10]: Thank you

**Comment by tak-li98** [comment]
Non-reviewer comment

**Review by atchu98 — APPROVED** [review]
Thanks for fixing the changes

**Review by Jayciee — APPROVED** [review]
Approved!

**Pull request merged**